sourced/ml/cmd_entries/args.py

```diff
@@ -18,9 +18,9 @@ def _get_help_string(self, action):
 
 def add_split_stem_arg(my_parser: argparse.ArgumentParser):
     my_parser.add_argument(
-        "--split", action="store_true",
-        help="Split identifiers based on special characters or case changes. For example split "
-             "'ThisIs_token' to ['this', 'is', 'token'].")
+        "--split", action="store_false",
```

Comment:
we should change the name to `--no-split`

Reply:
yes, `split or not to split - that's the question` - naming became confusing in this case)